src/components/Dropdown/Dropdown.js

```diff
@@ -60,7 +62,10 @@ class Dropdown extends React.PureComponent {
     return (
       <div className="Dropdown" data-element="dropdown" onClick={this.toggleDropdown}>
         <div className="items">
-          <div className="display-item">{this.getTranslatedContent(sortStrategy)}</div>
+          <div className="display-item">
+            <span>{this.getTranslatedContent(sortStrategy)}</span>
+            <Icon className="down-arrow" glyph="ic_arrow_drop_down_black_24px" />
```

Comment:
needed to add in icon so that the dropdown menu item width can stretch as long as the selected option.
Previously in the DOM, that wasn't the case. we just can't see it because we chose to have no border.

previously, the icon was being applied in the :after pseudo-selector